src/configure.ts

```diff
@@ -21,13 +18,41 @@ export async function jestPreviewConfigure(
 
   options.externalCss?.forEach((cssFile) => {
     const basename = path.basename(cssFile);
+
     // Avoid name collision
-    // Add `global` to let `jest-preview` server that we want to cache those files
-    const destinationBasename = `cache-${basename}`;
-    const destinationFile = path.join(
-      './node_modules/.cache/jest-preview-dom',
-      destinationBasename,
-    );
+    // Example: src/common/styles.css => cache-src___common___styles.css
+    const delimiter = '___';
+    const destinationBasename = `cache-${basename.replace('/', delimiter)}`;
+    const destinationFile = path.join(CACHE_FOLDER, destinationBasename);
+
+    // Create cache folder if it doesn't exist
+    if (!fs.existsSync(CACHE_FOLDER)) {
+      fs.mkdirSync(CACHE_FOLDER, {
+        recursive: true,
+      });
+    }
```

Comment:
I think we can add an `utils.ts` file which has `createCacheFolderIfNeeded()` or something like that. We have this logic in quite many places.